packages/lesswrong/lib/collections/tags/schema.ts

```diff
@@ -197,6 +197,12 @@ const schema: SchemaType<DbTag> = {
     optional: true,
     viewableBy: ['guests'],
   },
+  lastSubforumCommentAt: {
```

Comment:
If this won't be present on every tag, we should probably make it `nullable` to have more accurate code-gen